packages/elements/src/collapse/index.ts

```diff
@@ -173,16 +190,25 @@ export class Collapse extends BasicElement {
 
   protected render (): TemplateResult {
     return html`
-      <div part="header" role="heading" aria-level="${this.ariaLevel}">
+      <div part="header-wrapper" level="${this.level}">
         <slot name="header-left"></slot>
-        <ef-header part="header-toggle" level="${this.level}" @tap=${this.toggle} role="button" tabindex="0" aria-expanded="${this.expanded}" aria-controls="content">
-          ${this.header}
-          <ef-icon slot="left" icon="right" part="toggle"></ef-icon>
-        </ef-header>
+        <div part="header"
+             role="heading"
+             aria-level="${ifDefined(this.headingLevel || undefined)}">
+          <div part="header-toggle"
+               role="button"
+               tabindex="0"
+               aria-expanded="${this.expanded}"
+               aria-controls="content"
+               @tap=${this.toggle}>
+            <ef-icon icon="right" part="toggle"></ef-icon>
+            <span part="header-label">${this.header}</span>
+          </div>
+        </div>
         <slot name="header-right"></slot>
       </div>
       <div id="content" part="content" role="region" aria-labelledby="header-toggle">
```

Comment:
`aria-labelledby="header-toggle"` - id `header-toggle` does not exist

Reply:
aye, copied from somewhere

Reply:
Fixed